src/sass/index.css

```
@@ -1202,11 +1202,6 @@ p {
     margin: 0;
   }
```

Паддінг, що приблрали - все ок. 

Тут у .advantages__list  зайві
  ```
  display: inline-block;
  margin: 0 auto;
```
І на медіа-запиті тут можна залишити тільки display: flex; , все інше або параметри по замовчуванню, або нічого не змінює

```suggestion
@media screen and (min-width: 768px) {
  .advantages__list {
    display: flex;
 }
```

---

По зайвим рядкам - це хтось заліз у Вашу гілку і попрацював над своєю секцією, зараз розберемося хто. Він зможе перенести ці зміни в свою гілку і видалити з цієї  :)